libs/community/langchain_community/agent_toolkits/openapi/planner.py

```diff
@@ -371,7 +411,7 @@ def create_openapi_agent(
     tools = [
         _create_api_planner_tool(api_spec, llm),
         _create_api_controller_tool(
-            api_spec, requests_wrapper, llm, allow_dangerous_requests
+            api_spec, requests_wrapper, llm, allow_dangerous_requests, allow_operations
```

Comment:
```suggestion
            api_spec, requests_wrapper, llm, allow_dangerous_requests, allowed_operations
```

Reply:
@eyurtsev  Changes have been made to address this issue. Please see [commit](https://github.com/langchain-ai/langchain/pull/22962/commits/762f03d17bb5d55a390b101b4ebda160d8933394).